src/quo/components/list/item.cljs

```diff
@@ -4,7 +4,7 @@
             [quo.haptic :as haptic]
```

Comment:
could you pls rollback this file, we don't need to make changes in old files and designs